validatordiag/diag.go

```diff
@@ -0,0 +1,47 @@
+package validatordiag
```

Comment:
I believe this file shouldn't be recreated in preference of the new `helper/validatordiag/diag.go` 👍

Reply:
Removed.